gcampus/core/views/forms/measurement.py

```diff
@@ -117,6 +117,10 @@ class MeasurementEditView(
     def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
+    def get_context_data(self, **kwargs):
+        kwargs["measurement"] = self.object
```

Comment:
I prefere to use `kwargs.setdefault(...)` so that in theory `"measurement"` can be overwritten by subclasses.